MoyaLogger.swift

```diff
@@ -0,0 +1,67 @@
+import Foundation
+
+
+public struct MoyaLogger: Logger {
+    
+    private let loggerId = "Moya_Logger"
+    private let dateFormatString = "dd/MM/yyyy HH:mm:ss"
+    private let dateFormatter = NSDateFormatter()
+    
+    private var date: String {
+        
+        dateFormatter.dateFormat = dateFormatString
+        dateFormatter.locale = NSLocale(localeIdentifier: "en_US_POSIX")
+        return dateFormatter.stringFromDate(NSDate())
+    }
+    
+    public func logNetworkRequest(request: NSURLRequest) {
+        
+        let requestOutput = String(format: "%@: [%@] Request:  %@", loggerId, date, request.description)
```

Comment:
I really like your implementation here. One suggestion: where we are printing out multiple lines, does it make sense to include the same preamble on each line, when it's for the same request? What if we had something like:

```swift
let requestOutput = String(format: "%@: [%@] Request:  %@", loggerId, date, request.description)
print(requestOutput)
```

And then later, had something like:

```swift
let requestHeadersOutput = String(format: "        Request Headers:  %@", loggerId, date, headers)
print(requestHeadersOutput)
```

Does that make sense?